src/modules/payload_deliverer/payload_deliverer.cpp

```diff
@@ -125,20 +128,33 @@ void PayloadDeliverer::gripper_update(const hrt_abstime &now)
 {
 	if (!_gripper.is_valid()) {
 		// Try initializing gripper
-		initialize_gripper();
+		configure_gripper();
```

Comment:
Do you actually need this here? It's done in `init()` already.

Reply:
Nope definitely not!